packages/terra-functional-testing/src/services/wdio-visual-regression-service/modules/getTerraFormFactor.js

```diff
@@ -0,0 +1,22 @@
+import { TERRA_VIEWPORTS } from '../../../static-assets/viewports';
+/**
+ * Determines the Terra form factor to for the current viewport size.
+ *
+ * @param {Number} viewpointWidth - Current width of the viewpoint.
+ * @returns {String} - Terra form factor the current viewport with falls under.
+ */
+export default function getTerraFormFactor(viewpointWidth) {
+  // Default to enormous then check if the current viewport is a small form factor
+  let formFactor = 'enormous';
+
+  const viewportSizes = Object.keys(TERRA_VIEWPORTS);
+  for (let form = 0; form < viewportSizes.length; form += 1) {
+    const viewportName = viewportSizes[form];
+    if (viewpointWidth <= TERRA_VIEWPORTS[viewportName].width) {
+      formFactor = viewportName;
+      break;
```

Comment:
nit: Instead of redefining the formFactor this can just be returned. Falling back to enormous if no form factor was found (meaning it's larger than 1500).

```js
if (viewpointWidth <= TERRA_VIEWPORTS[viewportName].width) {
  return viewportName;
}
```

This is what the full function looks like (a few variable names were renamed in this example) :

```js
function getTerraFormFactor(width) {
  const viewports = Object.keys(TERRA_VIEWPORTS);

  for (let index = 0; index < viewports.length; index += 1) {
    const viewport = viewports[index];

    if (width <= TERRA_VIEWPORTS[viewport].width) {
      return viewport;
    }
  }

  return 'enormous';
}
```